ivadomed/training.py

```diff
@@ -22,11 +22,12 @@
 from ivadomed.loader import utils as imed_loader_utils
 from ivadomed.loader.balanced_sampler import BalancedSampler
 from ivadomed.keywords import ModelParamsKW, ConfigKW, BalanceSamplesKW, TrainingParamsKW, MetadataKW
+from ivadomed.random import set_seed
```

Comment:
In case you don't realize it, I really really appreciate absolute path import. 👍

Reply:
Okay. I do too, will ensure I continue doing that.